terraspace/app/stacks/peer/variables.tf

```diff
@@ -0,0 +1,27 @@
+variable "vpc" {
+  type = object({
+    name = string
+  })
```

Comment:
similar to our previous discussion, if the stack only needs the vpc name, lets make that the variable rather than a whole object.

Reply:
Not going to do that because VPC object is going to have new attributes based on another comments (cidr, private_subnets, etc...)